test/Core_utils.js

```diff
@@ -19,17 +19,37 @@
 //
 // ----------------------------------------------------------------------------
 
-var Core = artifacts.require("./Core.sol");
+const Core = artifacts.require("./Core.sol")
+  , SimpleToken = artifacts.require("./SimpleToken/SimpleToken.sol")
+  , Workers = artifacts.require("./Workers.sol")
+;
+
+const BigNumber = require('bignumber.js')
+;
+
 
 /// @dev Deploy 
 module.exports.deployCore = async (artifacts, accounts) => {
-	const registrar = accounts[1];
-	const chainIdOrigin = 3;
-	const chainIdRemote = 1410;
-	const openSTRemote = accounts[4];
+	const registrar = accounts[1]
+		, chainIdOrigin = 3
+		, chainIdRemote = 1410
+		, openSTRemote = accounts[4]
+		, valueToken   = await SimpleToken.new()
+    , admin = accounts[2]
```

Comment:
indentation, and grouping of accounts[] / types

Reply:
I assume grouping of accounts means like below:

registrar = accounts[1]
admin = accounts[2]
ops = accounts[3]
openSTRemote = accounts[4]

Can you please confirm?